docs/partC/tomancak/spatialtrackmatching.md

```diff
@@ -1,18 +1,153 @@
+
+The Spatial Track Matching plugin includes an option to plot cell division angles. Below is an example of such a plot:
+
+<img src="spatialtrackmatching/plot_cell_division_angle.png" width=600>
+
+The plot illustrates the cell division angles corresponding to matched cell divisions between two embryos.
+Up to timepoint 300, there are almost no division angles close to 90°,
+which indicates that the spatial track matching works reliably within this range.
+However, there are two cell division angles near 90° early on,
+which might suggest developmental differences between the two embryos.
+
+### Alignment of the Two Datasets
+
+The algorithm for finding correspondences relies on the angles between cell division directions in the embryos. Achieving a good alignment between the two embryos/datasets is critical for obtaining high-quality results. To facilitate this, three slightly different methods have been implemented for computing the alignment:
+
+#### 1. Fixed Spatial Registration Based on Root Cells
+
+This is the simplest approach and works well when the embryos remain relatively stationary during the recording. The alignment is based on the correspondence between the initial root cells (e.g., A ↔ A, B ↔ B, C ↔ C, D ↔ D). These root cells appear in multiple timepoints in the cell tracking data.
+
+For each root cell (e.g., cell A), there is not just one coordinate vector, but one for each timepoint before the cell divides. To simplify, the last coordinate vector before division is used. If there are four root cells (A, B, C, D), we extract one coordinate vector for each of these cells in embryo X and another set for the same cells in embryo Y. These vectors are then aligned by computing a transformation (translation, rotation, and scaling) that minimizes the squared distances between corresponding vectors. This transformation is used to align embryo Y with embryo X.
+
+#### 2. Dynamic Spatial Registration Based on Root Cells and Their Descendants
+
+This approach is more dynamic and works even if the embryos rotate during the recording. It starts with the correspondence between the initial root cells (e.g., A ↔ A, B ↔ B, C ↔ C, D ↔ D). 
+
+For each root cell (e.g., cell A), instead of extracting a fixed position vector. We extrac a position vector by timepoint which will be denoted as P(A,X,t), where A is the cell, X is the embrbryo and t is the timepoint. If the timepoint t is between the appearance of the cell A and there cell division P(A,X,t) will simple denote the position vector of the cell A. Afterwards when the cell divided into two daugther cells it will be the average position of the two daugther cells of cell A at that time point. When the daugter cell divide further it will simply be the average position of all the descendants of cell A at the time point in embryo A.
```

Comment:
```suggestion
For each root cell (e.g., cell A), instead of extracting a fixed position vector. We extract a position vector by timepoint which will be denoted as P(A,X,t), where A is the cell, X is the embryo and t is the timepoint. If the timepoint t is between the appearance of the cell A and its cell division, P(A,X,t) will simply denote the position vector of the cell A. Afterwards when the cell has divided into two daugther cells, it will be the average position of the two daugther cells of cell A at that time point. When the daughter cells divide further, it will simply be the average position of all the descendants of cell A at the time point in embryo A.
```